normandy/health/api/views.py

```diff
@@ -13,31 +13,41 @@
 
 
 _commit = None
+_tag = None
 
 
-def get_commit():
-    global _commit
+def get_version_info():
+    global _commit, _tag
```

Comment:
Man we could sure use a `cached_property` for functions.

Probably on the third time we do this.